content/docs/docs-nav.yaml

```diff
@@ -7,7 +7,7 @@ plugins:
 nav:
   - Getting started:
       - Installing the GitHub app: '/docs/getting-started/install-github-app/'
-      - Adding components: '/docs/getting-started/adding-components/'
+      - Add a catalog item: '/docs/getting-started/addcatalogitem'
```

Comment:
i think we should leave this out of the nav for this pull request. We can add it later. We should still be able to get to the page visiting the url.. but the users wont find it.